web/gds-user-ui/src/modules/dashboard/member/index.tsx

```diff
@@ -3,10 +3,14 @@ import MemberTable from './MemberTable';
 import { Trans } from '@lingui/macro';
 import Loader from 'components/Loader';
 import { useFetchMembers } from './hook/useFetchMembers';
+// import { mainnetMembersMockValue } from './__mocks__';
 
 const MemberPage: React.FC = () => {
   const { members, isFetchingMembers, error } = useFetchMembers();
 
+  // const vasps = mainnetMembersMockValue.vasps;
```

Comment:
Here is where I grabbed the mock data to send down from the parent component. I removed that change, but have kept this here for now.